Bugzilla/Elastic/Search.pm

```diff
@@ -90,8 +90,11 @@ sub data {
         die $@;
     }
     $self->_set_query_time($result->{took} / 1000);
+    my %stupid_name = (
+        changeddate => 'delta_ts',
+    );
+    my @fields = map { $stupid_name{$_} // $_ } @{ $self->fields };
     my (@ids, %hits);
```

Comment:
If you move this line to just after 92 ->_set_query_time, the diff reads more cleanly. But I like this arrangement better anyways because these are the loop variables and they should be declared closer to the loop than the one-time @fields stuff anyways.